tests/rancher/rancher_ui_selenium.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Web browser UI test for rancher container
```

Comment:
I suggest to explain in details here how this test differs from https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11807

Reply:
Thanks, I explained some aspects